framework/components/ecl-comments/ecl-comments.twig

```diff
@@ -49,17 +49,23 @@
       <p class="ecl-paragraph">{{ body }}</p>
     </div>
     {% if operations is defined %}
-    <div class="ecl-comment__operations">
+    <ul class="ecl-listing ecl-comment__operations">
```

Comment:
The actual class is `ecl-list ecl-list--inline`, not `ecl-listing`. ;)

"Listings" is more a layout component (https://ec-europa.github.io/europa-component-library/components/detail/ecl-listings), I know it can be confusing...